### PR TITLE
cannot evolve message contarcts that are not Provisioned

### PR DESCRIPTION
**Pull request by HrisyToteva**
closes dfds/cloudplatform/issues/2553

# Additional Review Notes

the frontend doesn’t allow you to evolve Messages Contracts that are not provisioned

image

### CODE REVIEW

File: src/src/pages/capabilities/KafkaCluster/MessageContract.module.css [hunk 1]
```diff
@@ -75,6 +75,7 @@
   animation-name: slidein;
   animation-duration: 0.5s;
   animation-timing-function: ease;
+  /* opacity: 0.25; */
```
Comment (kralle333) [inline src/src/pages/capabilities/KafkaCluster/MessageContract.module.css:78]: remove this :)

File: src/src/pages/capabilities/KafkaCluster/MessageContracts.js [hunk 2]
```diff
@@ -228,14 +226,20 @@ export default function MessageContracts({
             >
               {shownContracts.map((contract) => (
                 <option key={contract.id} value={contract.id}>
-                  Version {contract.schemaVersion}
+                  {`Version${contract.schemaVersion}${
```
Comment (kralle333) [inline src/src/pages/capabilities/KafkaCluster/MessageContracts.js:229]: you are missing a space between Version and the number

File: src/src/pages/capabilities/KafkaCluster/MessageContracts.js [hunk 3]
```diff
@@ -228,14 +226,20 @@ export default function MessageContracts({
...
+                    contract.status !== "Provisioned"
```
Comment (kralle333) [inline src/src/pages/capabilities/KafkaCluster/MessageContracts.js:230]: inside of MessageContracts.js we have an enum for the statuses. Use that instead
image

File: src/src/pages/capabilities/KafkaCluster/MessageContracts.js [hunk 4]
```diff
@@ -228,14 +226,20 @@ export default function MessageContracts({
...
+                  }`}
                 </option>
               ))}
             </SelectField>
             {onAddClicked && (
               <Button
                 variation="primary"
-                disabled={false}
+                disabled={
+                  selectedContract.status !== "Provisioned" ? true : false
```
Comment (kralle333) [inline src/src/pages/capabilities/KafkaCluster/MessageContracts.js:241]: same here, use the enum

File: src/src/pages/capabilities/KafkaCluster/MessageContracts.js [hunk 5]
```diff
@@ -278,15 +289,33 @@ export default function MessageContracts({
                 />
               }
             />
-            {selectedContract.description}
+            <Text
+              style={
+                selectedContract.status !== "Provisioned"
```
Comment (kralle333) [inline src/src/pages/capabilities/KafkaCluster/MessageContracts.js:294]: and here

**Review by kralle333 — CHANGES_REQUESTED** [review]
Great job! That was really quick. I have some smaller comments and then its ready to get in

File: src/src/pages/capabilities/KafkaCluster/MessageContracts.js (hunk 4 above)
Comment (HrisyToteva) [reply src/src/pages/capabilities/KafkaCluster/MessageContracts.js]: good point!

**Review by kralle333 — APPROVED** [review]
Thank you!

**Pull request merged**